spec/mode/list_spec.rb

```diff
@@ -3,6 +3,8 @@
 RSpec.describe 'The "list" mode', :type => :aruba do
   include_context "messages"
 
+  MOCK_VIEWER = SpecHelplers::Texdoc::MOCK_VIEWER
```

Comment:
Please use let block instead of defining constant to suppress `warning: already initialized constant MOCK_VIEWER`.

```suggestion
  let(:mock_viewer) { SpecHelplers::Texdoc::MOCK_VIEWER }
```